trunner/target/armv7a9.py

```diff
@@ -28,7 +28,11 @@ class ARMv7A9TargetRebooter(Rebooter):
     # TODO add text mode reboot
 
     def _reboot_soft(self):
-        self._reboot_hard()
+        self.host.set_reset(0)
+        time.sleep(0.5)
+        self.dut.clear_buffer()
+        self.host.set_reset(1)
+        time.sleep(0.25)
```

Comment:
It's already defined in the same form in the parent class, so you can just remove it just like it's done there: https://github.com/phoenix-rtos/phoenix-rtos-tests/blob/8c6d062c2fbd327e2664ec371fcf8bddf1135483/trunner/target/armv7m7.py#L21C14-L21C14

Reply:
@damianloew look closely on line 32